week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,45 @@
+/* 
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
+*/
+
+// write your code here
+
+var names = ["Rakesh", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
+var longNameThatStartsWithA = names.find(function(name) {
+  return name[0].includes("A") && name.length > 7;
+});
+
+console.log(longNameThatStartsWithA);
+
+/* EXPECTED OUTPUT */
+// "Alexandra"
+
 /* 
   You are given an array of test scores.
   Any result over 60 is a pass. 
   Using .find() (and any other array methods necessary), find the lowest pass score
 */
 
+/*
+Ignore! 
+
+*/
+
 var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+var filterResults = testScores.filter(filterScores).sort(function(num1, num2) {
+  return num1 - num2;
+});
+
+var findLowestPass = filterResults.find(function(num) {
```

Comment:
I guess this is something you want me to ignore? But I'll comment anyways, in case it confuses you, or someone else.
Don't think line 34 is necessary - you filtered out the scores below 60 and then sorted the array - at this point you can just get the first element in that array and you'll have your lowest passing score :)

Reply:
Hey!

I realized it wasn't necessary once I'd completed everything. However, I didn't change this and set it all to ignore as I was just using it as a playground of all the methods just to play about with them. I should have made this more clear!